aries/aries_vcx_core/Cargo.toml

```diff
@@ -13,8 +13,12 @@ vdr_proxy_ledger = ["credx", "dep:indy-vdr-proxy-client"]
 # Feature flag to allow legacy proof verification
 legacy_proof = []
 
+askar_wallet = ["dep:aries-askar", "dep:bs58", "dep:sodiumoxide"]
+
 [dependencies]
 agency_client = { path = "../misc/legacy/agency_client" }
+aries-askar = { git = "https://github.com/hyperledger/aries-askar.git", tag = "v0.3.0", optional = true }
```

Comment:
is there a reason for using git instead of the crates io version? also note crates.io is up to 0.3.1

https://crates.io/crates/aries-askar